eslint-warning-thresholds.json

```diff
@@ -27,15 +27,6 @@
     "@typescript-eslint/no-base-to-string": 1,
     "@typescript-eslint/no-shadow": 1
   },
-  "packages/design-system-react/src/components/AvatarAccount/AvatarAccount.tsx": {
-    "import-x/no-named-as-default-member": 1
-  },
-  "packages/design-system-react/src/components/AvatarBase/AvatarBase.tsx": {
-    "import-x/no-named-as-default-member": 1
-  },
-  "packages/design-system-react/src/components/AvatarFavicon/AvatarFavicon.tsx": {
-    "import-x/no-named-as-default-member": 1
-  },
```

Comment:
Automatically updating eslint threshold file using `yarn lint` which keeps track of our eslint warning violations the more we can remove from here the better